GraphQL.Net/GraphQLTypeBuilder.cs

```diff
@@ -129,6 +129,12 @@ public void AddAllFields()
                 _type.OwnFields.Add(CreateGenericField(prop));
         }
 
+        public void AddAllFields(Dictionary<string, string> aliasDictionary)
+        {
+            foreach (var key in aliasDictionary.Keys)
```

Comment:
Iterate the dictionary and operate on the key-value pairs to avoid doing a bunch of unnecessary lookups.